lib/utils.js

```diff
@@ -22,28 +22,38 @@ var utils = {
 		} else {
 			return {
 				statusCode: 200,
-				body: JSON.stringify({data: data})
+				body: JSON.stringify({ data: data })
 			};
 		}
 
 	},
 
-	generateToken: function(user) {
+	generateToken: function (user) {
 
-		return jwt.sign({user: user}, config.jwt.secret, {
+		return jwt.sign({ user: user }, config.jwt.secret, {
 			expiresIn: '2 days'
 		});
 	},
 
-	toAttributeUpdates: function(keyValues) {
-		return _.mapObject(keyValues, function(val, key) {
+	toAttributeUpdates: function (keyValues) {
+		return _.mapObject(keyValues, function (val, key) {
 			return {
 				Action: 'PUT',
 				Value: val
 			}
 		});
+	},
+    isEmpty: function (obj) {
```

Comment:
Don't need this function. Please use the underscore module to check. http://underscorejs.org/#isEmpty

var _ = require('underscore');
_.isEmpty()